docker-compose.build.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+services:
+    gradle:
+        build:
+            context: .
+            dockerfile: Dockerfile-gradle-with-tc
+        command: >
+            sh -c "cd /Octopus &&
+            ./gradlew -Dteamcity.distribution=/teamcity clean distZip &&
```

Comment:
shouldn't need the -Dteamcity.distribution (doesn't get used for anything anymore)

Reply:
Nice. It did _seem_ to work without it but i wasn't entirely sure so left it in to confirm with the readme. Taking this out will make it a little easier.